ui/app/src/components/ClustersManagement/ClustersManagement.tsx

```diff
@@ -96,7 +96,7 @@ export default function ClustersManagement() {
   return (
     <section>
       <GlobalAppToolbar
-        title={<FormattedMessage id="GlobalMenu.Cluster" defaultMessage="Cluster" />}
+        title={<FormattedMessage id="globalMenu.cluster" defaultMessage="Cluster" />}
```

Comment:
Didn't we have all of these with capitalization on the `G`? Why did you change it? Also why you changing clusters at all?